Corona-Warn-App/src/main/java/de/rki/coronawarnapp/diagnosiskeys/storage/KeyCacheDatabase.kt

```diff
@@ -45,6 +45,9 @@ abstract class KeyCacheDatabase : RoomDatabase() {
 
         @Update(entity = CachedKeyInfo::class)
         suspend fun updateDownloadState(update: CachedKeyInfo.DownloadUpdate)
+
```

Comment:
Use the generic @Update, so you can pass in a list of objects and make a batch update, which would be faster. Update the checked flag in the repo.

Reply:
I do not want to risk overwriting other fields and performance is not an issue (would only be less than 10 files)